assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -230,22 +230,27 @@ const baseActions = {
 		},
 		function* ( propertyID ) {
 			const registry = yield Data.commonActions.getRegistry();
+			const {
+				setPropertyCreateTime,
+				setSettings,
+				setWebDataStreamID,
+				updateSettingsForMeasurementID,
+			} = registry.dispatch( MODULES_ANALYTICS_4 );
+
+			setSettings( {
+				propertyID,
+				propertyCreateTime: 0,
+			} );
 
-			registry
-				.dispatch( MODULES_ANALYTICS_4 )
-				.setPropertyID( propertyID );
-			registry
-				.dispatch( MODULES_ANALYTICS_4 )
-				.setWebDataStreamID( WEBDATASTREAM_CREATE );
```

Comment:
Setting the stream to create here was the root cause as this causes the input to appear until the matched webdatastream is selected once loaded.